apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/StepBlockNode/libs/getCardMetadata/getCardMetadata.ts

```diff
@@ -106,13 +106,27 @@ export function getCardMetadata(
       hasMultipleActions
     }
   }
+
+  let imageTitle
+  let imageSubititle
+
+  if (priorityBlock?.__typename === 'ImageBlock') {
+    console.log('Priority image')
+    const width = priorityBlock?.width
+    const height = priorityBlock?.height
```

Comment:
Do you still need these? I think priorityBlock is not nullable at this point because of the typename check?
```suggestion
    const width = priorityBlock.width
    const height = priorityBlock.height
```